integration-guides/geotab.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Geotab Australia
+---
+
+To connect your Geotab telematics device integration with PathStack, you will need to retrieve your MyGeotab user email, the password you use to log into your MyGeotab account and the database name. 
+
+To find these credentials follow these next steps:
+
+<Steps>
+
+  <Step title = "Create a Geotab Service Account for PathStack">
+  Log in to MyGeotab account and navigate to (1) Administration > (2) Users, and click the (3) Add button. 
+  <img src="images/Integration-guides/geo_1_1.png" />
```

Comment:
this screenshot looks different to others

Reply:
At the moment, we do not have test account access to login and take screenshot of account home page.